docs/api/qiskit-ibm-provider/qiskit_ibm_provider.IBMBackend.md

```diff
@@ -10,11 +10,11 @@ python_api_name: qiskit_ibm_provider.IBMBackend
 
 <span id="qiskit_ibm_provider.IBMBackend" />
 
-`IBMBackend(configuration, provider, api_client, instance=None)`[GitHub](https://github.com/qiskit/qiskit-ibm-provider/tree/stable/0.7/qiskit_ibm_provider/ibm_backend.py "view source code")
+`IBMBackend(configuration, provider, api_client, instance=None)`[GitHub](https://github.com/qiskit/qiskit-ibm-provider/tree/stable/0.9/qiskit_ibm_provider/ibm_backend.py "view source code")
 
 Backend class interfacing with an IBM Quantum device.
 
-You can run experiments on a backend using the [`run()`](qiskit_ibm_provider.IBMBackend#run "qiskit_ibm_provider.IBMBackend.run") method. The [`run()`](qiskit_ibm_provider.IBMBackend#run "qiskit_ibm_provider.IBMBackend.run") method takes one or more [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit "(in Qiskit v0.45)") and returns an `IBMJob` instance that represents the submitted job. Each job has a unique job ID, which can later be used to retrieve the job. An example of this flow:
+You can run experiments on a backend using the [`run()`](#qiskit_ibm_provider.IBMBackend.run "qiskit_ibm_provider.IBMBackend.run") method. The [`run()`](#qiskit_ibm_provider.IBMBackend.run "qiskit_ibm_provider.IBMBackend.run") method takes one or more [`QuantumCircuit`](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit "(in Qiskit v0.46)") and returns an `IBMJob` instance that represents the submitted job. Each job has a unique job ID, which can later be used to retrieve the job. An example of this flow:
```

Comment:
Note that this URL is no longer relative. I think it's because intersphinx now uses docs.quantum.ibm.com in Provider. https://github.com/Qiskit/documentation/issues/493 will fix this.

I don't think we should block this PR on #493. But I'll bump it in priority. Relative links are better because our link checker can check them much more quickly. The good news is that the platform at least doesn't add the arrow suggesting it's an external link; i.e. it detects it's an internal link.

<img width="449" alt="Screenshot 2024-02-08 at 10 58 05 AM" src="https://github.com/Qiskit/documentation/assets/14852634/182614f2-de07-492f-bedf-bb5967a4f574">